tests/ensure_provider_tests.py

```diff
@@ -120,6 +120,7 @@
         - always
 
 - import_playbook: {test_playbook}
+  when: ansible_distribution_major_version != '9'
```

Comment:
I would suggest 
```
when: ansible_distribution_major_version | int < 9
```
that way it will work in rhel10, etc.

I don't think it is a good idea to use major_version without also checking `ansible_distribution` (unless you are checking `ansible_distribution` somewhere else in this code path), but doing it this way works well if we assume only rhel/centos/fedora

Reply:
Updated, thanks.